src/heretic/mn_menu.c

```diff
@@ -612,6 +612,31 @@ static void InitFonts(void)
     FontBBaseLump = W_GetNumForName(DEH_String("FONTB_S")) + 1;
 }
 
+// [crispy] Check if printable character is existing in FONTA/FONTB sets.
+enum {
+    big_font, small_font
+} fonttype_t;
+
+static const char MN_CheckValidChar (char ascii_index, int have_cursor)
+{
+    if ((ascii_index >= '[' + have_cursor && ascii_index <= '`') || ascii_index >= '{')
```

Comment:
Nitpick: Could you write `> 'Z'` and `< 'a'` and `> 'z'` here? This would make it clearer why these restrictions are there?

Reply:
Sure, will do. My idea of using `>=` was to point to exact chararacter, not to next one. Another thing is - we also have a [F_TextWrite](https://github.com/fabiangreffrath/crispy-doom/blob/master/src/heretic/f_finale.c#L150) font drawing function on finale screen, better I double check it too.